scenario/constraints/ProposalConstraint.ts

```diff
@@ -55,39 +55,34 @@ async function getAllPendingProposals(world: World, governor: string, admin: str
 
 export class ProposalConstraint<T extends CometContext, R extends Requirements> implements Constraint<T, R> {
   async solve(requirements: R, context: T, world: World) {
-    let solutions: Solution<T>[] = [];
-
     // Only run migration for mainnet scenarios
     if (await world.chainId() != 1) {
       return null;
     }
 
-    const governor = await context.getGovernor();
-
-    // XXX pull all pending proposals for the associated gov (chain)
-    // produce all the subsets and queue/execute each batch of them
-    // to form each solution
-    const pendingProposals = await getAllPendingProposals(world, governor.address, context.voters[0]);
+    return async function (ctx: T, wld: World): Promise<T> {
+      const governor = await ctx.getGovernor();
 
-    for (let proposal of pendingProposals) {
-      solutions.push(async function (context: T): Promise<T> {
+      // XXX pull all pending proposals for the associated gov (chain)
+      // produce all the subsets and queue/execute each batch of them
```

Comment:
We can remove this comment about subsets I think, and aren't we pulling them all now too?

Reply:
FYI cleaned this all up already in `clone-gov`, so I think next step is to merge that and reconcile with these other outstanding changes